frontends/web/src/locales/de/app.json

```diff
@@ -314,17 +300,63 @@
     "upgrade": "Aktualisieren"
   },
   "buy": {
+    "exchange": {
+      "bankTransfer": "Überweisung",
+      "bestDeal": "Bestes Angebot",
+      "creditCard": "Kreditkarte",
+      "fast": "Schnell",
+      "fee": "Gebühr",
+      "infoContent": {
+        "moonpay": {
+          "fees": {
+            "bankTransfer": "Banküberweisung: {{fee}}%",
+            "creditDebitCard": "Kredit-/Debitkarte: {{fee}}%",
+            "learnMore": "Erfahre mehr über Moonpay",
+            "title": "Gebühren"
+          },
+          "fullCurrenciesList": "Eine vollständige Liste der Währungen findest du hier",
+          "payment": {
+            "asteriskText": "* Nicht verfügbar für Einwohner der USA",
+            "bankTransfer": "Banküberweisung*",
+            "bankTransferDetails": {
+              "pix": "PIX (BR-Transaktionen nur in Brasilien)",
+              "sepa": "SEPA und SEPA Instant (EUR-Transaktionen nur in SEPA-Ländern)",
+              "uk": "UK Faster Payments (GBP-Transaktionen nur in Großbritannien)"
+            },
+            "creditDebitCard": "Kredit-/Debitkarte",
+            "creditDebitCardDetails": {
+              "cards": "Amex, Mastercard, Visa und Maestro"
+            },
+            "learnMore": "Weitere Details zu den Zahlungsmethoden findest du hier",
+            "title": "Zahlungsmethoden"
+          },
+          "supportedCurrencies": "Unterstützt alle gängigen Fiat-Währungen: USD, EUR, CHF und mehr."
+        },
+        "pocket": {
+          "fee": "Gebühr: {{fee}}%",
+          "learnMore": "Erfahre mehr über Pocket",
+          "supportedBankTransfers": "Unterstützt SEPA-Sofortüberweisungen in EUR und Sepa-Überweisungen in CHF"
```

Comment:
@benma this should change with https://github.com/digitalbitbox/bitbox-wallet-app/pull/1994

Reply:
updated

Reply:
thank you

Reply:
@benma I don't see them